x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -203,19 +210,28 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
     setCellProps: any;
   }) => {
     const cellValue = columnsData[rowIndex][columnId];
+    const actualCount = columnsData[rowIndex] && columnsData[rowIndex].actual_class_doc_count;
+    // @ts-ignore
+    let accuracy;
+
+    if (columnId !== ACTUAL_CLASS_ID && actualCount) {
+      accuracy = cellValue / actualCount;
+      // round to 2 decimal places without converting to string;
+      accuracy = Math.round(accuracy * 100) / 100;
+      accuracy = `${Math.round(accuracy * 100)}%`;
+    }
```

Comment:
Good catch, @walterra  - for now I'll just add the type to the initial `accuracy` variable to avoid a larger change. Happy to move it out of the file in a follow-up. 👍

Reply:
Updated in 422c7aacdfc9d129a7877af2d9f1c8ebede15f80